src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -93,7 +93,7 @@ public void setDocuments(List<BundleDocument> documents) {
     }
 
     public String getFileName() {
-        return fileName == null || fileName.isEmpty() ? bundleTitle + ".pdf" : fileName;
+        return fileName == null || fileName.isEmpty() ? bundleTitle : fileName;
```

Comment:
This only implemented the change if bundleTitle was used.

Reply:
If the end user has specified a filename shouldn't we just do what they tell us to do? I can see a lot of people putting in file.pdf and it coming out file.pdf.pdf